modular_skyrat/master_files/code/modules/client/preferences/mutant_parts.dm

```diff
@@ -392,95 +392,199 @@
 
 /// IPC Screens
 
-/datum/preference/toggle/mutant_toggle/ipc_screen
-	savefile_key = "ipc_screen_toggle"
-	relevant_mutant_bodypart = "ipc_screen"
-
 /datum/preference/choiced/mutant_choice/ipc_screen
 	savefile_key = "feature_ipc_screen"
-	relevant_mutant_bodypart = "ipc_screen"
-	type_to_check = /datum/preference/toggle/mutant_toggle/ipc_screen
+	main_feature_name = "IPC Screen"
+	category = PREFERENCE_CATEGORY_FEATURES
+	relevant_mutant_bodypart = MUTANT_SYNTH_SCREEN
 	default_accessory_type = /datum/sprite_accessory/screen/none
+	should_generate_icons = TRUE
+	generate_icons = TRUE
+	crop_area = list(11, 22, 21, 32) // We want just the head.
+	greyscale_color = DEFAULT_SYNTH_SCREEN_COLOR
 
-/datum/preference/tri_color/ipc_screen
-	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+/datum/preference/choiced/mutant_choice/ipc_screen/is_part_enabled(datum/preferences/preferences)
+	return TRUE
```

Comment:
Remind me, what does this do, again?

Reply:
In the mutant base class, it reads `type_to_check`'s value, which isn't used for IPC screens, cause they're race-specific, and I don't feel like making that mutant choice work across all races. Ditto for chassis and head choices, along with the colours.